### DISCUSSION TITLE
Show newest Issue as cover for series.

### DISCUSSION THREAD

**Discussion by XendoxShade23** [Kareadita/Kavita#3247] [Ideas]
### Idea Description

It would be cool if there was an option to have the cover for a series automatically update to the cover of the newest issue.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

6 months.

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
This is part of this feature request #2629 please upvote there.
